java-ast/addon/src/main/java/org/jboss/windup/ast/java/ReferenceResolvingVisitor.java

```diff
@@ -111,7 +113,7 @@ private void resolveCurrentTypeNames(CompilationUnit compilationUnit)
             {
                 typeLine = typeLine.substring(typeDeclaration.getJavadoc().toString().length());
             }
-            classReferences.add(new ClassReference(fqcn, ResolutionStatus.RESOLVED, TypeReferenceLocation.TYPE,
```

Comment:
lol, apparently I am a little off today. Yes, that is my change and wasn't removing what I thought it was. I just misread it.